src/main/cliapp/src/service/TableStateService.js

```diff
@@ -239,13 +239,17 @@ export function getModFormFields(table) {
 	return modFormFields;
 };
 
-export function getDefaultTableState(tableName, columns, defaultColumnWidth, defaultVisibleColumns) {
+export function getDefaultTableState(tableName, columns, defaultColumnWidth, obsoleteDefaultFilter, defaultVisibleColumns) {
+	
```

Comment:
I know this ticket is specifically for the obsolete filter, but I think it would be simpler and more flexible to allow the calling table pass in a defaultFilters object and set that in the filter field on the returned object.